src/com/rath/rathbot/cmd/disc/ReportCmd.java

```diff
@@ -17,33 +35,76 @@ public String getCommandDescription() {
         + "You will need the member's internal ID. Get it using the \"uid <Discord username>\" command. "
         + "The uid command also requires a direct message."
         + "Note that this is NOT their nickname for the server, but the username you see when you open their profile."
-        + "For example, Kami's Discord username is \"Danni293\".";
+        + "For example, Kami's Discord username is \"Loli no Kami\".";
   }
   
   @Override
   public String getCommandUsage() {
-    return "rb! report <uid> <Reason..>";
+    return "rb! report <uid> <reason..>";
   }
   
   @Override
   public int permissionLevelRequired() {
     return RBCommand.PERM_MINIMAL;
   }
   
+  @Override
+  public boolean requiresDirectMessage() {
+    return true;
+  }
+  
   @Override
   public boolean executeCommand(final IMessage msg, final String[] tokens, final int tokenDepth) {
     
-    System.out.println("Executing report");
+    IUser issuedUser = msg.getAuthor();
+    
+    // Ensures at least minimum valid arguments used.
+    if (tokens.length < 4) {
+      RathBot.sendDirectMessage(issuedUser, "Syntax Error! Usage: rb! report <uid> <reason>");
```

Comment:
Instead of copying the syntax, you could've concatenated this string with a call to getCommandUsage(). It's best to have one single message that you edit rather than multiple, which is why I'm going to migrate messages to a single class or package.